library.properties

```diff
@@ -0,0 +1,10 @@
+name=Queue
+version=0.0.1
+author=Steven de Salas <steven@desalasworks.com>
+maintainer=Steven de Salas <steven@desalasworks.com>
+sentence=Generic C++ circular queue
+paragraph=Works for any datatype and desired length
+category=Data Storage
+url=https://github.com/ifreecarve/arduino_ci/SampleProjects/DoSomething
```

Comment:
The URL should be https://github.com/sdesalas/Arduino-Queue.h

Reply:
Whoops! Good catch